tests/testthat/test-get_rcode.R

```diff
@@ -7,17 +7,21 @@ testthat::test_that("get_rcode returns header only for empty chunks", {
   # removing the Date line from the header as the seconds may be different
   # in the two strings
   testthat::expect_identical(
-    strsplit(r_code_from_chunks, "\n")[[1]][c(1:5, 7:14)],
-    strsplit(r_code_from_header, "\n")[[1]][c(1:5, 7:14)]
+    strsplit(r_code_from_chunks, "\n")[[1]][c(1:5, 7:15)],
+    strsplit(r_code_from_header, "\n")[[1]][c(1:5, 7:15)]
   )
 })
 
 testthat::test_that("get_rcode returns code from chunks at the end", {
   ch <- teal.code::chunks_new()
   teal.code::chunks_push(id = "test", chunks = ch, quote(a <- 1))
+
+  r_code_from_chunks <- get_rcode(chunks = ch)
+  r_code_from_header <- sprintf("\n\n%s\n\na <- 1", paste(get_rcode_header(), collapse = "\n"))
+
   testthat::expect_identical(
-    get_rcode(chunks = ch),
-    sprintf("\n\n%s\n\na <- 1", paste(get_rcode_header(), collapse = "\n"))
+    strsplit(r_code_from_chunks, "\n")[[1]][c(1:5, 7:17)],
+    strsplit(r_code_from_header, "\n")[[1]][c(1:5, 7:17)]
```

Comment:
on my system should be up to 15 not 17

Reply:
```
> length(strsplit(r_code_from_chunks, "\n")[[1]])
[1] 15
> length(strsplit(r_code_from_header, "\n")[[1]])
[1] 15
```

Reply:
updated the tests to use `length` so that it is consistent on any platform.